pages/apis/rest_api/jobs.md.erb

```diff
@@ -34,10 +34,10 @@ curl -X PUT "https://api.buildkite.com/v2/organizations/{org.slug}/pipelines/{pi
       "runnable_at": null,
       "started_at": null,
       "finished_at": null,
-      "retried": "true",
-      "retried_in_job_id": "b63254c0-3271-4a98-8270-7cfbd6c2f14e",
-      "retries_count": "1",
-      "retry_type": "automatic",
+      "retried": false,
+      "retried_in_job_id": null,
+      "retries_count": 1,
+      "retry_type": null,
```

Comment:
I think it was good to leave the example with "automatic"  instead of "null" It was added in #1387

Reply:
Oops. 

I'd really like to find the time to look into automating the API bodys.